unattended_installer/install_functions/checks.sh

```diff
@@ -147,6 +147,71 @@ function checkArguments() {
 
 }
 
+function checkFirewalls() {
+
+
+    firewallsList=( "iptables"
+                    "nft"
+                    "ufw"
+                    "firewall-cmd")
+    
+    portsTCPLists=( "1514"
+                    "1515"
+                    "1516"
+                    "514"
+                    "55000"
+                    "9200"
+                    "9300"
+                    "9400"
+                    "443")
+
+    for command in "${firewallsList[@]}"; do
+
+        if [ -n "$(command -v $command)" ]; then
+            logger_cert "The $command command is present on this system. This could affect the correct communication between Wazuh components. We will proceed to try to validate firewall rules that may affect the processes and report what is found."
+            firewallstatus='true'
+
+            if [ $command == "iptables" ]; then
```

Comment:
Use switch case instead of 4 if command == value.

Reply:
Done